build/dependencies.props

```diff
@@ -4,6 +4,7 @@
     <InternalAspNetCoreSdkVersion>2.0.0-*</InternalAspNetCoreSdkVersion>
     <ResponseCompressionVersion>1.1.0-*</ResponseCompressionVersion>
     <AspNetCoreLabsVersion>0.3.0-*</AspNetCoreLabsVersion>
```

Comment:
There are other "Labs" packages, for example https://dotnet.myget.org/feed/aspnetcore-ci-dev/package/nuget/Microsoft.AspNetCore.Buffering

I'm not sure the status of them, or if we care about ever referencing them, but I'd rather keep ti around just in case.